docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -149,6 +149,10 @@ This page allows you to do the following:
 
   - To delete a table from the replication, click **... > Remove Table**. This removes both the table and its index tables from replication. If you decide to remove an index table from the replication group, it does not remove its main table from the replication group.
 
+    {{<note title="Dropping tables">}}
```

Comment:
```suggestion
    {{<note title="Before dropping tables">}}
```

Reply:
just feel the title can sound more like a warning so the user definitely reads it.